<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSpatialObjectsModule</class>
 <widget class="qSlicerWidget" name="qSlicerSpatialObjectsModule">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="SpatialObjectsTableDisplay" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text" stdset="0">
      <string>Simple Display</string>
     </property>
     <property name="collapsed" stdset="0">
      <bool>false</bool>
     </property>
     <property name="collapsedHeight" stdset="0">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_table">
      <item row="0" column="0">
       <widget class="qMRMLSpatialObjectsTreeView" name="SpatialObjectsTreeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="listenNodeModifiedEvent" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" stdset="0">
      <string>Advanced Display</string>
     </property>
     <property name="collapsed" stdset="0">
      <bool>true</bool>
     </property>
     <property name="collapsedHeight" stdset="0">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="TractDisplayModesTabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="LineTab">
         <attribute name="title">
          <string>Line</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="qSlicerSpatialObjectsWidget" name="LineDisplayWidget" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_13">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="TubeTab">
         <attribute name="title">
          <string>Tube</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="qSlicerSpatialObjectsWidget" name="TubeDisplayWidget" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_15">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="GlyphTab">
         <attribute name="title">
          <string>Glyph</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="qSlicerSpatialObjectsWidget" name="GlyphDisplayWidget" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="qSlicerSpatialObjectsGlyphWidget" name="GlyphPropertiesWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_16">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_14">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerSpatialObjectsWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerSpatialObjectsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerSpatialObjectsGlyphWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerSpatialObjectsGlyphWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSpatialObjectsTreeView</class>
   <extends>QTreeView</extends>
   <header>qMRMLSpatialObjectsTreeView.h</header>
   <slots>
    <signal>currentNodeChanged(vtkMRMLNode*)</signal>
    <slot>setCurrentNode(vtkMRMLNode*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerSpatialObjectsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>GlyphDisplayWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerSpatialObjectsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>TubeDisplayWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerSpatialObjectsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>LineDisplayWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerSpatialObjectsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>GlyphPropertiesWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerSpatialObjectsModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SpatialObjectsTreeView</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpatialObjectsTreeView</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>qSlicerSpatialObjectsModule</receiver>
   <slot>setSpatialObjectsNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>currentNodeChanged(vtkMRMLNode*)</signal>
  <slot>setSpatialObjectsNode(vtkMRMLNode*)</slot>
 </slots>
</ui>
